19/README.md

```diff
@@ -0,0 +1,128 @@
+```
+shortname: 19/tm_stress
```

Comment:
[BEP-19](https://github.com/bigchaindb/BEPs/pull/45) is taken too. I as I [previuosly](https://github.com/bigchaindb/BEPs/pull/52#discussion_r196709563) you can use BEP-20